### DISCUSSION TITLE
Copy & pasted example in User Guide

### DISCUSSION THREAD

**Discussion by eisthf** [fastapi/fastapi#11870] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# I just wanted to mention that one example code is mistakenly copy and pasted. I am not sure if this is the right place to mention document issue in.
```

### Description

The example code in "FastAPI > Learn > Tutorial - User Guide > Using lists instead of sets" is just copy & past of the right above and does not reflect the intended point.

### Operating System

Windows

### FastAPI Version

0.111.0

### Pydantic Version

2.7.4

### Python Version

3.11.5

**Comment by YuriiMotov** [comment, marked as answer]
It's not a copy, there are differences:
```
    response_model_include={"name", "description"},
...
@app.get("/items/{item_id}/public", response_model=Item, response_model_exclude={"tax"})

```
VS
```
    response_model_include=["name", "description"],
...
@app.get("/items/{item_id}/public", response_model=Item, response_model_exclude=["tax"])
```

Notice, that curly braces were changed to square brakets
